frontend/src/components/VendorIcon.vue

```diff
@@ -65,17 +62,12 @@ export default {
       }
       return undefined
     },
-    getHeight () {
-      return this.height
-    },
-    getWidth () {
-      if (!this.width && !this.height) {
-        return 20
-      }
-      return this.width
-    },
     isMdiIcon () {
       return startsWith(this.value, 'mdi-')
+    },
+    iconStyle () {
```

Comment:
Use classes if possible. Static attributes should be static not bound. Use Object Syntax (inline if possible) not String syntax https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax-1

Reply:
done